.github/workflows/continous-integration-docker.yml

```
@@ -299,6 +299,52 @@ jobs:
         env:
           ZEBRA_TEST_LIGHTWALLETD: '1'
 
+  # Test that Zebra works using the default config with the latest Zebra version
+  #
+  # Also check that $ZEBRA_CONF_PATH works
+  test-configuration-file:
+    name: Test Zebra configuration file
```

Nitpick: I had some trouble finding the right job in the summary view

```suggestion
    name: Test Zebra default Docker config file
```